test/modules/sungeun/init/printModuleInitOrder.good

```diff
@@ -11,7 +11,6 @@ Initializing Modules:
      ChapelStringLiterals
     MemConsistency
     Atomics
-    AtomicsCommon
```

Comment:
Same reason as the last - the autoCopy now does deep copies instead of changing reference counts. AtomicsCommon has the reference counting routines, which are no longer called.